src/components/structures/RoomStatusBar.js

```diff
@@ -231,9 +196,7 @@ module.exports = React.createClass({
     },
 
     _shouldShowConnectionError: function() {
-        // no conn bar trumps unread count since you can't get unread messages
-        // without a connection! (technically may already have some but meh)
-        // It also trumps the "some not sent" msg since you can't resend without
```

Comment:
This is still the case though?

Reply:
No, as the unread count isn't shown in this component anymore...
The unread count is now shown on the JumpToBottomButton independent of whether the RoomStatusBar is shown.

Reply:
Yeah, I rephrased that, see the added line below.

Reply:
I'm blind. Thank you :D